cocotb/regression.py

```diff
@@ -617,21 +618,19 @@ def _create_test(function, name, documentation, mod, *args, **kwargs):
     Returns:
         Decorated test function
     """
+
+    @wraps(function)
     async def _my_test(dut):
-        await function(dut, *args, **kwargs)
+        await function(cocotb.top, *args, **kwargs)
```

Comment:
I know `cocotb.top` is equivalent (right now), but I think it's better to leave the handle passing to the Regression Manager.

```suggestion
        await function(dut, *args, **kwargs)
```